### ISSUE TITLE
mj-font, mj-attributes or mj-title fail to render in the browser

### ISSUE THREAD

**Issue by ianchadwick** [mjmlio/mjml#395]
Only just started playing so it's very likely I'm doing something wrong but when I try any of the mj-head components it fails to render.

```
import {mjml2html} from 'mjml';

const html = mjml2html(`
<mjml>
  <mj-head>
    <mj-font name="Anton" href="https://fonts.googleapis.com/css?family=Anton" />
  </mj-head>
  <mj-body>
    <mj-container>
      <mj-text font-family="Anton">Hello there!</mj-text>
    </mj-container>
  </mj-body>
</mjml>`);
```
Throws: Uncaught TypeError: Cannot read property 'name' of undefined

This is the same for mj-title and mj-attributes.

Changing how the attributes are accessed fixes it... feels like this can't be broken for everyone though - maybe it is?
```js
# from...
fonts.push({ name: el.attribs.name, url: el.attribs.href });

#to...
fonts.push({ name: el.getAttribute('name'), url: el.getAttribute('href') });
```

**Comment by iRyusa** [comment]
Hi  @ianchadwick !
Can you provide your Node & MJML versions ?

**Comment by ianchadwick** [comment]
Hello!

I'm using node v4.5.0 and here are the mjml versions:
```
├─┬ mjml@2.3.3
│ ├── mjml-button@2.0.10
│ ├─┬ mjml-cli@2.3.2
│ ├── mjml-column@2.0.10
│ ├── mjml-container@2.0.10
│ ├─┬ mjml-core@2.3.2
│ ├── mjml-divider@2.0.11
│ ├── mjml-group@2.0.4
│ ├── mjml-head-attributes@2.0.7
│ ├── mjml-head-font@2.0.1
│ ├── mjml-head-title@2.0.1
│ ├── mjml-hero@2.0.10
│ ├── mjml-html@2.0.9
│ ├── mjml-image@2.0.10
│ ├─┬ mjml-invoice@2.0.11
│ ├── mjml-list@2.0.9
│ ├── mjml-location@2.0.10
│ ├── mjml-navbar@2.0.8
│ ├── mjml-raw@2.0.9
│ ├── mjml-section@2.0.10
│ ├── mjml-social@2.0.10
│ ├── mjml-spacer@2.0.8
│ ├── mjml-table@2.0.9
│ └── mjml-text@2.0.9

**Comment by iRyusa** [comment]
It's perfectly fine on my setup, can you try upgrade to node 6.X to see if it's an node related issue ?

**Comment by iRyusa** [comment]
Oh... as @lohek mentioned to me, I didn't seen the "in the browser" .

On server side we use `cheerio` to parse the MJML file, but on the browser side we're using jQuery to do dom related things. 

Thing is, we've started an helper to unify both world called `dom.js` but it looks like `getChildren` doesn't work the same.

When we find a fix for that, we should be able to write for both browser & server : 
```javascript
fonts.push({ name: el.getAttribute('name'), url: el.getAttribute('href') });
```

**Comment by iRyusa** [comment]
I'm closing this one, update about browser side MJML will be available in #274
